R/MakeKRICharts.R

```diff
@@ -56,5 +59,28 @@ MakeKRICharts <- function(lData = NULL) {
     )
 
 
+  # Continuous Charts -------------------------------------------------------
+    if (!is.null(lStackedSnapshots)) {
+
+      number_of_snapshots <- length(unique(lStackedSnapshots$rpt_site_kri_details$snapshot_date))
```

Comment:
Could we just combine `lStackedSnapshots` and `lData`? Could the stacked version of `rpt_site_kri_details`  just (optionally) be added to lData? Is `stacked$rpt_site_kri_details` really needed? Will look a bit more ...